src/components/modules/ApplePayButtonView.res

```diff
@@ -0,0 +1,40 @@
+open ReactNative
+
+type nativeSyntheticEvent<'a> = {nativeEvent: 'a}
+
+type prop = {
+  buttonType?: string, // (buy, setUp, inStore, donate, checkout, book, subscribe)default-> plain
+  buttonStyle?: string, // (white, whiteOutline)default->black
```

Comment:
There are predefined set of values that are allowed for buttonType and buttonStyle, then maybe define them as [polymorphic variants](https://rescript-lang.org/docs/manual/latest/polymorphic-variant#type-declaration), so that invalid value cannot be passed.